src/data/fintype/basic.lean

```diff
@@ -692,6 +692,8 @@ instance : fintype bool := ⟨⟨tt ::ₘ ff ::ₘ 0, by simp⟩, λ x, by cases
 instance units_int.fintype : fintype (units ℤ) :=
 ⟨{1, -1}, λ x, by cases int.units_eq_one_or x; simp *⟩
 
+@[simp] lemma int.units_univ : (finset.univ : finset (units ℤ)) = {1, -1} := rfl
```

Comment:
This would match the name of the fintype instance:
```suggestion
@[simp] lemma units_int.univ : (finset.univ : finset (units ℤ)) = {1, -1} := rfl
```
Renaming the instance above is also fine by me.